spectral/ruleset.yml

```diff
@@ -568,4 +520,4 @@ rules:
       function: pattern
       functionOptions:
         match: >-
-          ^((https):\/\/)([a-z-]+)([.]+)([a-z]?)bettercloud.com\/[a-z-]+\/v[1-9]+$
+          ^((https):\/\/)(x-holding|x-staging|api)([.]+)(bettercloud.com|devbettercloud.com)\/[a-z-]+\/v[1-9]+$
```

Comment:
I think I'd rather leave it generic in case we add a URL for the mock server or change our nonprod environments. Just replace [a-z]? with [a-z]*